config.template.yml

```diff
@@ -516,3 +516,16 @@ notifier:
   ##   sender: admin@example.com
   ##   host: smtp.gmail.com
   ##   port: 587
+
+identity_providers:
+  oidc:
+    hmac_secret: this_is_a_secret_abc123abc123abc
+    issuer_private_key: |
+      --- KEY START
+      --- KEY END
+    clients:
+      - id: myapp
+        secret: this_is_a_secret
+        policy: this_policy
```

Comment:
We need to document this is meant to be?

Reply:
Yeah we need to decide what to do with ACL related things. I have a few ideas. For example if we add this at the start we can use it to define a policy level (one_factor or two_factor) for  the client. Then later we can add some customization to the authorization module to account for OIDC.

Some special method that only requires the subject (user, groups, network) and the OIDC callback. So it wouldn't consider resources. It may be beneficial to add a new area of the ACL's like so:


```
policy: two_factor
callbacks:
- http://myapp.com/oauth2/callback
```

Reply:
Done, though I've not wired up the functionality yet. We should discuss what v1 will look like as far as policies.